user/stub/user.go

```diff
@@ -33,11 +33,12 @@ type Authentication struct {
 type JoinReason uint32
 
 const (
-	JoinReasonSchool     JoinReason = 0
-	JoinReasonWork       JoinReason = 1
-	JoinReasonFascinated JoinReason = 2
-	JoinReasonFriend     JoinReason = 3
-	JoinReasonOther      JoinReason = 4
+	JoinReasonJOIN_REASON_UNSPECIFIED JoinReason = 0
+	JoinReasonJOIN_REASON_SCHOOL      JoinReason = 1
+	JoinReasonJOIN_REASON_WORK        JoinReason = 2
+	JoinReasonJOIN_REASON_FASCINATED  JoinReason = 3
+	JoinReasonJOIN_REASON_FRIEND      JoinReason = 4
+	JoinReasonJOIN_REASON_OTHER       JoinReason = 5
```

Comment:
kalau tujuan ngeletakin JoinReason di depan biar sesuai konteks, harusnya ga perlu. karena JOIN_REASON sudah mewakili. Masak mau kayak java -,-
```suggestion
	JOIN_REASON_UNSPECIFIED JoinReason = 0
	JOIN_REASON_SCHOOL      JoinReason = 1
	JOIN_REASON_WORK        JoinReason = 2
	JOIN_REASON_FASCINATED  JoinReason = 3
	JOIN_REASON_FRIEND      JoinReason = 4
	JOIN_REASON_OTHER       JoinReason = 5
```

Reply:
Ini ubahnya jangan disini, ubah di repo proto2http.

Reply:
jadinya diubah?

Reply:
iya

Reply:
ubahnya jangan disini woi

Reply:
> ubahnya jangan disini woi

 iyaa wkwk

Reply:
https://github.com/kodiiing/proto2http/pull/7